core/primitives/src/offchain.rs

```diff
@@ -54,7 +54,7 @@ impl TryFrom<u32> for StorageKind {
 
 impl From<StorageKind> for u32 {
 	fn from(c: StorageKind) -> Self {
-		c as u8 as u32
+		u32::from(c as u8)
```

Comment:
isn't u32::from panicking at runtime if overflow ? this change the behavior. but might be better indeed

Reply:
I believe that this should be equivalent. `From::from` is defined to be infailable.